probe.h

```diff
@@ -29,8 +29,8 @@
 // Probe pin initialization routine.
 void probe_init();
 
-// Returns probe pin state.
-uint8_t probe_get_state();
+// Returns probe pin state. Triggered = true. Called by gcode parser and probe state monitor.
+#define probe_get_state() (!(PROBE_PIN & PROBE_MASK))
```

Comment:
It's called by the time-sensitive `probe_state_monitor`, so I wanted to inline it to avoid function call overhead.  It is called one other place, but it's so tiny that the size is comparable to the function call it replaces.